addons/point_of_sale/migrations/13.0.1.0.1/noupdate_changes.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version='1.0' encoding='utf-8'?>
 <odoo>
-  <record id="pos_config_main" model="pos.config">
+<!--  <record id="pos_config_main" model="pos.config">
     <field name="picking_type_id"/>
-  </record>
+  </record>-->
   <record id="product_product_consumable" model="product.product">
     <field name="default_code">DISC</field>
-    <field name="image" type="base64" file="point_of_sale/static/img/product_product_49-image.jpg"/>
+    <field name="image_1920" type="base64" file="point_of_sale/static/img/product_product_49-image.jpg"/>
```

Comment:
Is this image file missing from this PR?

Reply:
deleted